docs/one-wire-devices.md

```diff
@@ -0,0 +1,150 @@
+
+<h2 id="protocol">What is a One-Wire Protocol</h2>
+
+The One-Wire protocol is a communication protocol that establishes a low-speed communication bridge between One-Wire Devices and processors/micro-controllers over a single data line. With a single data line as a communication methodology, the wiring process is tremendously simplified and the number of pins required to bridge the connection is also reduced. 
+
+One Wire protocol follows a master-slave architecture with each bus allowing for one master, in this case, an Omega2 board, and a slave device like the DS18B20 temperature sensor.
+
+<h2 id="connect">Connect a One-Wire Device to the Omega2</h2>
+
+This section covers a three-stage process for establishing the physical wiring connection of your One-Wire device to GPIO19, where the latter will serve the role of the One-Wire master.
+
+### Stage 1: Identify the Connectors
+
+One-Wire devices have three available connectors/wires as illustrated in the table below: 
+
+| Wire            | Connection                                                      |
+|-----------------|-----------------------------------------------------------------|
+| VCC             | Connect this wire to the 3.3V power source on the Omega2 board. |  
+| GND             | Connect this wire to the ground pin on the Omega2 board.        |  
+| DQ              | Connect this wire to the GPIO19 pin on the Omega2 board.        | 
+
+![one wire device circuit diagram](../static/img/one-wire-device-circuit-diagram-1.png)
+
+**NOTE:** Please refer to your specific One-Wire device's datasheet to identify the pins and determine the recommended voltage. 
+
+**TIP:** Building these connections is relatively easy if you have an Expansion, Power, or Arduino Dock since they all expose the Omega’s GPIOs. 
+
+**WARNING:** Some One-Wire devices will require a pull-up resistor on the Data line (DQ). For example, the popular DS18B20 temperature sensor requires a 4.7 kΩ pull-up resistor on the DQ to operate properly.
+
+### Stage 2: Connect Power & Ground
+
+Connect the power and ground wires belonging to your One-Wire device to the corresponding power and ground pins on the Omega2 board. 
+
+### Stage 3: Connect the DQ
+
+Connect the DQ from your One-Wire device to the GPIO19 pin on the Omega2 board. 
+
+<h2 id="enable">Enable One-Wire Device Support</h2>
+
+
+Once the physical connections are made, install dts overlay package to register a one-wire master in Linux associated with GPIO19 to communicate with the one-wire slave device.  
+
+**Step 1:**  Install the `onion-dt-overlay-w1-gpio` package using opkg
+
+<pre>
+<code>opkg update
+opkg install onion-dt-overlay-w1-gpio</code>
+</pre>
+
+**Step 2:** After the package is successfully installed, GPIO19 will act as a One-Wire Master. 
+
+<pre>
+<code>/sys/devices/w1_bus_master1</code>
+</pre>
```

Comment:
For code blocks, please use the built-in markdown code blocks: https://docusaurus.io/docs/markdown-features/code-blocks.

Again, we should avoid using html unless absolutely necessary